Cargo.toml

```diff
@@ -10,24 +10,35 @@ name = "cursive"
 readme = "Readme.md"
 repository = "https://github.com/gyscos/Cursive"
 version = "0.3.1"
+build = "build.rs"
 
 [build-dependencies]
 skeptic = "0.6"
 
+[features]
+cursive_ncurses = ["ncurses"]
```

Comment:
Probably because "Features and dependencies cannot have the same name: `ncurses`". Also, you'll need a `no_default_features` there :v